Tests/PetstoreConsumerTests/Test_Server.swift

```diff
@@ -231,6 +231,35 @@ final class Test_Server: XCTestCase {
         )
     }
 
+    
+    func testCreatePet_withIncorrectContentType() async throws {
+        client = .init(
+            createPetBlock: { input in
+                XCTFail("The handler should not have been called")
```

Comment:
The CI is complaining about this, we need to return Never here, for example like this.
```suggestion
                XCTFail("The handler should not have been called")

                fatalError("Unreachable")
```

Reply:
Oh sorry, after applied the suggestion, test looks passed locally.